client/main.go

```diff
@@ -10,20 +10,79 @@ import (
 	"os"
 	"os/exec"
 	"regexp"
+	"strings"
 	"syscall"
 	"time"
 
 	"github.com/richo/roving/types"
 )
 
-type Server struct {
+type RovingServer struct {
 	hostport string
 }
 
+type FuzzCommand interface {
```

Comment:
We possibly don't even need `FuzzCommand` (since it's only used for its `cmd()` method), but I think it's kind of nice to have different params directly accessible (eg. `fuzzerId`, `memLimit`). Wdyt?